examples/app-basic/.github/workflows/ci.yml

```
@@ -62,14 +64,14 @@ jobs:
 
       - name: Test backend
         if: steps.check-for-backend.outputs.has-backend == 'true'
-        uses: magefile/mage-action@v3
+        uses: magefile/mage-action@6f50bbb8ea47d56e62dee92392788acbc8192d0b
```

You should add a comment next to each hash with the git tag for the action. This will allow dependabot to update it. Like so:

```suggestion
        uses: magefile/mage-action@6f50bbb8ea47d56e62dee92392788acbc8192d0b # v3
```